routes/users.js

```diff
@@ -0,0 +1,11 @@
+import dotenv from "dotenv";
+import express from "express";
+import { authenticateByGoogle, signupByEmail } from "../controllers/users.js";
+
+dotenv.config({ path: "./config.env" });

```

Comment:
Why is this needed?

Reply:
This allows to use environment variables. I am moving this line to controller to avoid confusion.